### PR TITLE
[DEM-927] Implement the LockInStimulus interface.

### PR DESCRIPTION
**Pull request by qSaevar**

### CODE REVIEW

File: src/qtt/measurements/acquisition/lock_in_stimulus.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+""" Provides the LockInStimulus"""
+from qilib.configuration_helper import InstrumentAdapterFactory
+from qilib.utils import PythonJsonStructure
+
+
+class LockInStimulus:
```
Comment (peendebak) [inline src/qtt/measurements/acquisition/lock_in_stimulus.py]: Rename to UHFLIstimulus? And also update the filename? The class is clearly UHFLI only.

File: src/qtt/measurements/acquisition/lock_in_stimulus.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+""" Provides the LockInStimulus"""
```
Comment (peendebak) [inline src/qtt/measurements/acquisition/lock_in_stimulus.py]: Remove this comment? It does not add a lot of information

File: src/qtt/measurements/acquisition/lock_in_stimulus.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        """ Apply configuration to the UHFLI.
+
+        Args:
+            configuration: The configuration for the UHFLI.
+
+        """
+        self._adapter.apply(configuration)
+
+    def set_demodulation_enabled(self, channel: int, is_enabled: bool) -> None:
+        """ Equivalent to enable data transfer in the demodulators section of the Lock-in tab
```
Comment (peendebak) [inline src/qtt/measurements/acquisition/lock_in_stimulus.py]: First line of comment should be a single line

File: src/qtt/measurements/acquisition/lock_in_stimulus.py (hunk 1 above)
Comment (qSaevar) [reply src/qtt/measurements/acquisition/lock_in_stimulus.py]: done

File: src/qtt/measurements/acquisition/lock_in_stimulus.py (hunk 2 above)
Comment (qSaevar) [reply src/qtt/measurements/acquisition/lock_in_stimulus.py]: done

File: src/qtt/measurements/acquisition/lock_in_stimulus.py (hunk 3 above)
Comment (qSaevar) [reply src/qtt/measurements/acquisition/lock_in_stimulus.py]: ✔️

**Pull request merged**